stock_lot_analytic/models/analytic_mixin.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Quartile Limited (https://www.quartile.co)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models
+
+
+class AnalyticMixin(models.AbstractModel):
+    _inherit = "analytic.mixin"
+
+    def _format_distribution(self, dist):
```

Comment:
It could be a good idea to move this to `analytic_mixin_analytic_account` if we can make this generic enough.